website/content/docs/concepts/plugins/csi.mdx

```diff
@@ -30,9 +30,14 @@ plugin. Jobs can claim storage volumes from AWS Elastic Block Storage
 (EBS) volumes, GCP persistent disks, Ceph, Portworx, vSphere, etc. The
 Nomad scheduler will be aware of volumes created by CSI plugins and
 schedule workloads based on the availability of volumes on a given
-Nomad client node. A list of available CSI plugins can be found in the
-[Kubernetes CSI documentation][csi-drivers-list]. Any of these plugins
-should work with Nomad out of the box.
+Nomad client node.
+
+A list of available CSI plugins can be found in the [Kubernetes CSI
+documentation][csi-drivers-list]. Any of these plugins should work with Nomad
+out of the box, but it's some plugin vendors have implemented their plugins in a
```

Comment:
```suggestion
However, it is possible a plugin vendor has implemented their plugin 
```

Reply:
"it's some"? What was I even typing there? 😊